tools/unicode_properties_parse/format_width_estimate_intervals.py

```diff
@@ -140,7 +130,7 @@ def get_width(str: str):
         line = line.strip()
         if line and not line.startswith("#"):
             match = LINE_REGEX.fullmatch(line)
-            assert match, "invalid line"
+            assert match, line # invalid line

```

Comment:
This will print line details when things goes wrong.
(Actually, I've found that the regex has been slightly outdated due to some added whitespaces in  `EastAsianWidth-15.1.0.txt` (in https://www.unicode.org/Public/UCD/latest/ucd/EastAsianWidth.txt; which recently updated))
(I think we don't have to update the regex eagerly in this pr as it still works for 15.0.0)